pkg/apis/extensions/v1alpha1/types_controlplane.go

```diff
@@ -92,7 +94,9 @@ type Purpose string
 
 const (
 	// Normal triggers the ControlPlane controllers for the shoot provider.
+	// Deprecated: This value is no longer necessary and will be removed in a future release.
```

Comment:
We could also add deprecated tag to the Purpose type: https://github.com/gardener/gardener/blob/2f496e961e8bb552043281f68826d91759527d7c/pkg/apis/extensions/v1alpha1/types_controlplane.go#L93